easybuild/easyconfigs/c/CellBender/CellBender-0.2.2-foss-2021b.eb

```
@@ -0,0 +1,70 @@
+easyblock = 'PythonBundle'
+
+name = 'CellBender'
+version = '0.2.2'
```

since we're depending on `CUDA`, we should use the appropriate `versionsuffix`. @fizwit note that you should update the easyconfig filename to match
```suggestion
name = 'CellBender'
version = '0.2.2'
versionsuffix = '-CUDA-%(cudaver)s'
```